plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -720,6 +707,31 @@ SecRule REQUEST_FILENAME "@endsWith /settings/users" \
     ctl:ruleRemoveTargetByTag=OWASP_CRS;ARGS:newuserpassword,\
     ctl:ruleRemoveTargetByTag=OWASP_CRS;ARGS:password"
 
+# Nextcloud administrator setting/changing a user's password
+SecRule REQUEST_FILENAME "@rx /ocs/v[0-9]\.php/cloud/users(?:/[^/]+)?$" \
+    "id:9508501,\
+    phase:1,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
+    ctl:ruleRemoveTargetByTag=OWASP_CRS;ARGS:json.password,\
+    ctl:ruleRemoveTargetByTag=OWASP_CRS;ARGS:json.value"
+
+# Fix Nextcloud Cookie FPs
+SecAction \
+    "id:9508510,\
+    phase:1,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
+    ctl:ruleRemoveTargetById=942450;REQUEST_COOKIES:oc_sessionPassphrase,\
+    ctl:ruleRemoveTargetById=942210;REQUEST_COOKIES:oc_sessionPassphrase,\
+    ctl:ruleRemoveTargetById=942450;REQUEST_COOKIES:nc_session_id,\
+    ctl:ruleRemoveTargetById=942450;REQUEST_COOKIES:nc_token,\
+    ctl:ruleRemoveTargetById=942450;REQUEST_COOKIES:ocddjoynjqgd"
```

Comment:
Is this value really stable??

Reply:
hm... it looks to be unique to each Nextcloud instance. It's a PL-2 FP so it's not a big deal but still annoying.